doc/content/integrations/node-red/setup/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Setup"
+description: ""
+weight: 1
+---
+
+This section describes how to setup a Node-RED server and prepare to connect it to {{% tts %}}. 
+
+## Requirements
+
+1. [Install Node-RED](https://nodered.org/docs/getting-started/local)
+
+>Note: Node-RED v1.0.6 is current at time of writing and is used in this guide.
+
+## Setup
+
+Run Node-RED by using the following command in the terminal:
+
+```bash
+$ node-red 
+```
```

Comment:
We don't have to add anything to https://nodered.org/docs/getting-started/local; remove these instructions. There are better ways to run than this, as mentioned on that page and references above